test/packaging/windows/msi.tests.ps1

```diff
@@ -49,6 +49,26 @@ Describe -Name "Windows MSI" -Fixture {
                 $useMu = Get-ItemPropertyValue -Path $key -Name UseMU -ErrorAction SilentlyContinue
             } catch {}
 
+            if (!$useMu) {
+                $useMu = 0
+            }
+
+            return $useMu
+        }
+
+        function Set-UseMU {
+            param(
+                [int]
+                $Value
```

Comment:
It looks like `$Value` is never used and the value is always set to `0`.

Reply:
fixed